cli/config.go

```diff
@@ -50,19 +87,129 @@ func ParseConfig() {
 		log.Fatal(err)
 	}
 
-	if config.Offset != 0 {
-		Offset = config.Offset
+	if config.Filter.Offset != "" {
+		offset, err := strconv.ParseUint(config.Filter.Offset, 10, 64)
+		if err != nil {
+			log.Println("Error converting Offset to uint64:", err)
```

Comment:
use mainflux logger here and in other instances

Reply:
I have addressed the comments